x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/findings_flyout.test.tsx

```diff
@@ -66,4 +75,20 @@ describe('<FindingsFlyout/>', () => {
       getAllByText(mockFindingsHit.resource.id);
     });
   });
+
+  it('should allow pagination with next', async () => {
+    const { getByTestId } = render(<TestComponent />);
+
+    userEvent.click(getByTestId('pagination-button-next'));
+
+    expect(onPaginate).toHaveBeenCalledWith(1);
+  });
+
+  it('should allow pagination with previous', async () => {
+    const { getByTestId } = render(<TestComponent flyoutIndex={1} />);
+
+    userEvent.click(getByTestId('pagination-button-previous'));
```

Comment:
is this test id ours? or does it come with the component?
If its ours, lets import it from the shared `test_subject` file

Reply:
This is from Eui, I don't think it is worth moving to the shared file